hoomd/SnapshotSystemData.cc

```diff
@@ -39,6 +39,40 @@ void SnapshotSystemData<Real>::replicate(unsigned int nx, unsigned int ny, unsig
     pair_data.replicate(n, old_n);
     }
 
+template<class Real> void SnapshotSystemData<Real>::wrap()
+    {
+    vec3<double> frac;
+    vec3<double> wrapped;
+    vec3<double> pos;
+    vec3<double> fin;
+    int3 img;
+
+    for (unsigned int i = 0; i < particle_data.size; i++)
+        {
+        pos = particle_data.pos[i];
+        frac = global_box.makeFraction(particle_data.pos[i]);
+        wrapped = vec3<double>(
+            fmod(fmod(frac.x, 1.0) + 1.0, 1.0),
+            fmod(fmod(frac.y, 1.0) + 1.0, 1.0),
+            fmod(fmod(frac.z, 1.0) + 1.0, 1.0)
+            );
+        fin = global_box.makeCoordinates(wrapped);
+        particle_data.pos[i] = fin;
```

Comment:
Remove the unnecessary variable `fin`. It's generally better to use long variable names like `final_position` wherever a variable is needed.
```suggestion
        particle_data.pos[i] = global_box.makeCoordinates(wrapped);
```

Reply:
Thanks for catching. Aimed to be temporary while I still had the print statements